docs/reference/field-types/checkbox.md

```diff
@@ -0,0 +1,66 @@
+# `checkboxes`
```

Comment:
@abea @boutell Is this our lonely plural guy we didn't make singular?

Reply:
The fields weren't singularized like modules, if that's what you're thinking of. This might be an odd duck, but it's not because of that.

Reply:
This is plural because it is *always* an array, which is a distinction worth making, although its singular equivalent happens to be named `boolean`. (This one really ought to be called `booleans`! We don't usually name our fields after UI choices for representing them. But that's not something to hold this PR for.)